plugins/newsql/newsql.c

```diff
@@ -1876,15 +1879,18 @@ static CDB2QUERY *read_newsql_query(struct dbenv *dbenv,
                          gbl_nid_dbname, NULL, 0, 0) != 1) {
             newsql_error(clnt, "Client certificate authentication failed.",
                          CDB2ERR_CONNECT_ERROR);
+            logmsg(LOGMSG_ERROR, "client certificate authentication failed\n");
```

Comment:
No need to log this down. `sslio_accept` already does it.

Reply:
sslio_accept passes NULL for error (out) string. Also, don't see sslio_accept_or_connect() logging any info.